internal/autocert/manager.go

```diff
@@ -92,6 +92,8 @@ func (mgr *Manager) getCertMagicConfig(options *config.Options) (*certmagic.Conf
 			return nil, fmt.Errorf("config: failed caching cert: %w", err)
 		}
 	}
+	// use the same logger as certmagic
+	acmeTemplate.Logger = mgr.certmagic.Logger
```

Comment:
this is not thread safe. It modifies a global variable.

Reply:
Thanks.  Fixed with a local copy rather than getting rid of the global.   The tests relied on the global acmeTemplate for some setup and I broke them on the first attempt.